README.md

```diff
@@ -1,8 +1,63 @@
-# codacy-public-template
+# codacy-metrics-detekt
 
-Template repository for new public repositories. 
+This is the docker engine we use at Codacy to have [Detekt](https://github.com/arturbosch/detekt) metrics support.
 
-## GitHub actions
+## Build
 
-This repository has the common GitHub actions that we want to have accross all of our public repositories.
-They should be kept at `.github/workflows`
+### Requirements
+
+* [Docker](https://www.docker.com/)
+* [sbt](https://www.scala-sbt.org/)
+* Java 8+
+
+### Steps
+
+```bash
+sbt docker:publishLocal
+```
+
+## Run
+
+### Requirements
+
+* [Docker](https://www.docker.com/)
+
+### Steps
+
+```bash
+docker run -it -v <SRC_DIR>:/src <DOCKER_NAME>:<DOCKER_VERSION>
+```
+
+* `<SRC_DIR>` should be the directory where your project is.
+* `<DOCKER_NAME>` should be the docker image name you created in the Build step
+* `<DOCKER_VERSION>` should be the docker image version from the Build step
+
+Example:
+
+```bash
+docker run -it -v $PWD:/src codacy-metrics-detekt:0.1.0
```

Comment:
I'd remove the version here:
```suggestion
docker run -it -v $PWD:/src codacy-metrics-detekt
```

Reply:
The version here helps illustrate the line above
```bash
docker run -it -v <SRC_DIR>:/src <DOCKER_NAME>:<DOCKER_VERSION>
```